### ISSUE TITLE
uTox opens new window offscreen (windows 7)

### ISSUE THREAD

**Issue by Nsheppard01** [notsecure/uTox#934]
I normaly use a two monitor setup, but right now I do not have both monitors with me. Whenever I try to open a new instance of uTox however, it creates the new window offscreen, presumably where the other monitor would be.

I can move the window over by shift clicking the taskbar icon and then selecting the move option. But this is inconvenient and took some figuring out. This also happens each time my computer restarts and I have to open a new instance of uTox.

I have close to tray and start in tray both set to Yes. No idea if this might effect it.

**Comment by GrayHatter** [comment]
@Nsheppard01 I think I know how to fix this but until I can get around to it... https://xkcd.com/1479/

**Comment by Nsheppard01** [comment]
Now that I have both of my screens this isn't a problem for me. But Tox still always spawns new windows in the far right monitor no matter what. Again this isn't a problem because that's where I'd drag it anyways, but it's bad behavior, a window should always open in the screen it was opened from.

**Comment by Kyokou** [comment]
@Nsheppard01 
I disagree with your assertion that a window should open in the screen it was opened from. Programs should open, by default, on the [1] monitor that the user has set as their default 'parent' on the first open. The next time the program is opened, it should remember it's previous location and open to that location. 

In the event that location no longer exists (ie. user has removed a monitor) it returns to the default [1] position as if it has been opened for the first time.

From a user experience perspective - it removes the need for a user to constantly move the window to their desired location. 

I'm not sure if the UX is different in Linux, but this is the expected behavior of Windows programs - to the extent some developers will get flack for not conforming to the unofficial standard of "Leave it where I put it."
